test/integration/user_can_create_account_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+# As a visitor
+#     When I visit "/"
+#     Then I should see a link for "Login"
+#     And when I click link "Create Account"
+#     Then my current page should be "/dashboard"
+#     And I should see a message in the navbar that says "Logged in as SOME_USER"
+#     And I should see my profile information
+#     And I should not see a link for "Login"
+#     And I should see a link for "Logout"
+
+class UserCanCreateAccountTest < ActionDispatch::IntegrationTest
+  test "user can create account and sees profile" do
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.